accounting/accounting_cli.py

```diff
@@ -0,0 +1,148 @@
+#!/usr/bin/env python3
+
+###############################################################
+# Copyright 2020 Lawrence Livermore National Security, LLC
+# (c.f. AUTHORS, NOTICE.LLNS, COPYING)
+#
+# This file is part of the Flux resource manager framework.
+# For details, see https://github.com/flux-framework.
+#
+# SPDX-License-Identifier: LGPL-3.0
+###############################################################
+import sqlite3
+import argparse
+import time
+
+import pandas as pd
+
+from accounting import accounting_cli_functions as aclif
+
+
+def main():
+
+    conn = sqlite3.connect("FluxAccounting.db")
+
+    parser = argparse.ArgumentParser(
+        description="""
+        Description: Translate command line arguments into
+        SQLite instructions for the Flux Accounting Database.
+        """
+    )
+    subparsers = parser.add_subparsers(help="sub-command help",)
+
+    subparser_view_user = subparsers.add_parser(
+        "view-user", help="view a user's information in the accounting database"
+    )
+    subparser_view_user.add_argument(
+        dest="view_user", type=str, help="username", metavar=("USER_NAME")
+    )
+
+    subparser_add_user = subparsers.add_parser(
+        "add-user", help="add a user to the accounting database"
+    )
+    subparser_add_user.add_argument(
+        "--user_name",
```

Comment:
For all of these long options with the double-dash, I recommend switching from underscores to hyphens.  It is the style recommended by the GNU style guide [[1]](https://stackoverflow.com/questions/1253679/should-command-line-options-in-posix-style-operating-systems-be-underscore-style)

```suggestion
        "--user-name",
```

Although in this specific case, isn't it normally a single word? `username` [[2]](https://english.stackexchange.com/questions/43436/username-user-name-or-user-name) (This is the definition of a nitpick so feel free to ignore this second half of the suggestion).

Reply:
additional food for thought, `--user` instead for consistency to other tools.  BUT the other tools sometimes take a userid too.

Reply:
Yeah, using `--user` allows the option to take either user name or userid.

Reply:
in this case wasn't clear if this tool could take a userid

Reply:
Sorry, I commented without even looking at the code. Bad @grondo!
However, I meant that it leaves an opening to extend this option in the future to take either username or userid...